app/scripts/filters/resources.js

```diff
@@ -34,7 +34,9 @@ angular.module('openshiftConsole')
       "buildPod":                 ["openshift.io/build.pod-name"],
       "jenkinsBuildURL":          ["openshift.io/jenkins-build-uri"],
       "jenkinsLogURL":            ["openshift.io/jenkins-log-url"],
-      "jenkinsStatus":            ["openshift.io/jenkins-status-json"]
+      "jenkinsStatus":            ["openshift.io/jenkins-status-json"],
+      "idledAt":                  ["idling.alpha.openshift.io/idled-at"],
+      "unidleTarget":             ["idling.alpha.openshift.io/unidle-targets"]
```

Comment:
Im not sure atm, but I added it for reference since it was part of the idle PR.  I'm guessing its the annotation that would go on the endpoint & tell us what DC/RC is idled.  Since we have the annotation on the DC/RC directly, maybe its nothing the UI would ever use & we can drop it.